<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsFormObjectRefAssignment</class>
 <widget class="QDialog" name="CwmsFormObjectRefAssignment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auswahl f체r Objektzuordnung</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QRadioButton" name="m_pqrbContainer">
     <property name="text">
      <string>Container ausw채hlen</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="m_pqlvObjectLists">
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Schl체ssel</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_pqrbWql">
     <property name="text">
      <string>WQL</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_pqgbWql">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>WQL</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Aktualisierung abh채ngig von Member</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="m_pqcbMembers"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="m_pqteWql"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>201</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbCancel">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../WMSadmin/resources/emresources.qrc">
         <normaloff>:/icons/cancel24.png</normaloff>:/icons/cancel24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbOK">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../WMSadmin/resources/emresources.qrc">
         <normaloff>:/icons/ok24.png</normaloff>:/icons/ok24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Alt+O</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../WMSadmin/resources/emresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pqrbContainer</sender>
   <signal>clicked()</signal>
   <receiver>CwmsFormObjectRefAssignment</receiver>
   <slot>ContainerClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqrbWql</sender>
   <signal>clicked()</signal>
   <receiver>CwmsFormObjectRefAssignment</receiver>
   <slot>WQLClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbOK</sender>
   <signal>clicked()</signal>
   <receiver>CwmsFormObjectRefAssignment</receiver>
   <slot>OKClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>600</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbCancel</sender>
   <signal>clicked()</signal>
   <receiver>CwmsFormObjectRefAssignment</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ContainerClickedSlot()</slot>
  <slot>WQLClickedSlot()</slot>
  <slot>OKClickedSlot()</slot>
 </slots>
</ui>
